install.sh

```diff
@@ -565,23 +545,44 @@ _uninstall() {
     # Kill all sync jobs and remove sync folder
     [ -z "${SKIP_SYNC}" ] && command -v "${SYNC_COMMAND_NAME}" 2>| /dev/null 1>&2 && {
         "${SYNC_COMMAND_NAME}" -k all 2>| /dev/null 1>&2 || :
-        rm -rf "${INFO_PATH}"/sync "${INSTALL_PATH:?}"/"${SYNC_COMMAND_NAME}"
+        chmod -f u+w "${INSTALL_PATH}/${SYNC_COMMAND_NAME}"
+        rm -rf "${INFO_PATH}"/sync "${INSTALL_PATH}/${SYNC_COMMAND_NAME}"
```

Comment:
Codacy found an issue: [Use "${var:?}" to ensure this never expands to / .](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6201392)